sysl2/sysl/validate/validate.go

```diff
@@ -153,7 +150,7 @@ func (v *Validator) validateViews() {
 				v.compareTuple(t.Tuple, resolvedType.GetTuple(),
 					getAttrNames(resolvedType.GetTuple().GetAttrDefs()), viewName, typeName)
 			default:
-				fmt.Println("[validate.validateViews] Unhandled grammar type")
+				logrus.Println("[validate.validateViews] Unhandled grammar type")
```

Comment:
This should be a more specific error classification, like `Error` or `Warn`.

Reply:
Changed to warn